contrib/openpgp-card-ssh-agent/template.py

```diff
@@ -0,0 +1,18 @@
+pkgname = "openpgp-card-ssh-agent"
+pkgver = "0.2.3"
+pkgrel = 0
+build_style = "cargo"
+hostmakedepends = ["cargo", "pkgconf"]
+makedepends = ["rust-std", "pcsc-lite-devel"]
+depends = ["pcsc-ccid"]
+pkgdesc = "Simple ssh-agent backed by OpenPGP card authentication keys"
```

Comment:
```suggestion
pkgdesc = "SSH-agent backed by OpenPGP card authentication keys"
```

(in general try to avoid adjectives, fluff, the language it's written in unless it's for a library/sole disambiguation, etc)